package.json

```
@@ -111,7 +112,9 @@
     "prettier": "^2.6.2",
     "react": "^18.0.0",
     "react-color": "^2.19.3",
+    "react-docgen-typescript": "2",
```

Could we add a minor and patch version here?
```suggestion
    "react-docgen-typescript": "^2.2.2",
```